lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -21,11 +47,17 @@ def initialize(serializer, options = {})
         def serializable_hash(options = nil)
           options ||= {}
 
-          if serializer.respond_to?(:each)
-            serializable_hash_for_collection(options)
-          else
-            serializable_hash_for_single_resource(options)
-          end
+          hash =
+            case
```

Comment:
Why the extra line for a `case` statement? I generally like it, but this really looks like an `if...else` situation to me.